### ISSUE TITLE
Formatting error

### ISSUE THREAD

**Issue by Hoffenbar** [marktext/marktext#2962]
If I open a md with `***` at the very last line, this show as is even if not in source mode:

![image](https://user-images.githubusercontent.com/46227007/151698074-fe90230c-be71-46d5-91ad-c381175ef879.png)

Here is the same passage in source code:

![image](https://user-images.githubusercontent.com/46227007/151698092-322e9c70-8df4-4fe7-a2a1-a3a6c45463d4.png)

This already happens with 1.6.3

Sometimes, swithing forth an back an scrolling, the preview is then correct...

**Comment by YalandHong** [comment]
How to reproduce this issue? Are you sure it's an error?

By default, MarkText puts your cursor to the bottom when you open a file. And `***` won't be rendered as a line if your cursor is on it.

![div](https://user-images.githubusercontent.com/52960068/151702478-46fe6760-6796-4191-b548-dbb573072e06.gif)

**Comment by Hoffenbar** [comment]
Ok. I will test further, at least it's irritating :-)

**Comment by Hoffenbar** [comment]
You are right. When I move the cursor, the `***` immediately turns into `-----------------------....`

**Comment by Jocs** [comment]
Thank you for the report! This is actually the intended WYSIWYG behavior of MarkText. When your cursor is positioned on a thematic-break block (`***`, `---`, etc.), the editor displays the raw markdown syntax so you can edit it directly. As soon as you move the cursor to any other block, `***` is immediately rendered as a horizontal rule. This was confirmed in the thread by both a contributor and yourself — "When I move the cursor, the `***` immediately turns into `---...`".

Because MarkText now uses a new editor engine (`@muyajs/core`) that preserves the same intentional cursor-on-block display behavior (see `inlineRenderer/renderer/index.ts` `getClassName`), there is no bug to fix here. Closing as by-design.
